appinventor/blocklyeditor/src/backpack.js

```diff
@@ -47,7 +47,7 @@ goog.require('AI.Blockly.Util');
  * Backpack's flyout workspace.
  * @constructor
  */
-Blockly.Backpack = function(targetWorkspace, opt_options) {
+Blockly.Backpack = function (targetWorkspace, opt_options) {
```

Comment:
it seems like you IDE is reformatting stuff that we don't want to change; otherwise the diffs are difficult to read.